packages/react/src/avatar/Avatar.css.ts

```diff
@@ -69,14 +67,23 @@ export const fallback = recipe({
     size: "full",
     textTransform: "uppercase",
   },
+});
 
+export const icon = recipe({
+  base: [
+    {
+      h: "auto",
+    },
+  ],
   variants: {
     size: {
-      xs: { px: "4" },
-      sm: { px: "6" },
-      md: { px: "8" },
-      lg: { px: "10" },
-      xl: { px: "20" },
+      xs: style({ width: "12px" }),
+      sm: style({ width: "14px" }),
+      md: style({ width: "16px" }),
+      xl: style({ width: "20px" }),
+      "5xl": style({ width: "30px" }),
```

Comment:
let's use sprinkle props for these? and shift the sizes slightly to match the existing tokens for now:

```suggestion
      xs: { w: "12" },
      sm: { w: "12" },
      md: { w: "16" },
      xl: { w: "20" },
      "5xl": { w: "32" },
```